Library/Homebrew/dev-cmd/merge-homebrew.rb

```diff
@@ -18,7 +23,7 @@ def git_merge
     puts "End   commit: #{end_sha1}"
 
     safe_system "git", "checkout", "master"
-    safe_system "git", "merge", "homebrew/master", "-m", "Merge branch homebrew/master into linuxbrew/master"
+    safe_system "git", "merge", "--no-ff", "homebrew/master", "-m", '"Merge branch homebrew/master into linuxbrew/master"'
```

Comment:
Doesn't the latter result in double quotes in the git commit message, or am I wrong?